.changeset/quick-sloths-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-shell': patch
```

Comment:
This is the only package that needs to be released, because of the `test-utils`. The other packages were also updated, but only the test files.